charts/azuremonitor-containers-geneva/values.yaml

```diff
@@ -17,7 +17,7 @@ image:
   repository: mcr.microsoft.com/azuremonitor/containerinsights/ciprod
   tag: "3.1.15"
   pullPolicy: IfNotPresent
-  agentVersion: "azure-mdsd-1.17.1"
+  agentVersion: "azure-mdsd-1.28.11"
```

Comment:
@pfrcks  looking at the attached screenshots, does the first screenshot mean that there is memory increase with MSI mode with latest mdsd version?

Reply:
its only a slight increase so can be ignored. will observe more closely in ci/cd clusters